packages/manager/modules/pci/src/projects/project/storages/containers/container/addUser/addUser.html

```diff
@@ -65,7 +65,7 @@
             data-type="submit"
             data-size="l"
             data-on-click="$ctrl.addUserStorage()"
-            data-disabled="$ctrl.isLoading || !$ctrl.selectedUser"
+            data-disabled="$ctrl.isLoading || ($ctrl.addUserStep === 0 ? !$ctrl.selectedUser : !$ctrl.selectedRole)"
```

Comment:
Could the condition be extracted to specific method? It will ease reading conditions

Reply:
Done.